br/use-br-command-line-tool.md

```diff
@@ -22,36 +22,26 @@ This is a complete `br` command:
 {{< copyable "shell-regular" >}}
 
 ```shell
-br backup full --pd "${PDIP}:2379" -s "local:///tmp/backup"
+`br backup full --pd "${PDIP}:2379" -s "s3://backup-data/2022-01-30/"`
 ```
 
 Explanations for the above command are as follows:
 
 * `backup`: the sub-command of `br`.
 * `full`: the sub-command of `backup`.
 * `-s` (or `--storage`): the option that specifies the path where the backup files are stored.
-* `"local:///tmp/backup"`: the parameter of `-s`. `/tmp/backup` is the path in the local disk where the backed up files of each TiKV node are stored.
+* `"s3://backup-data/2022-01-30/"`: the parameter of `-s`, indicating that backup data is stored to the `2022-01-30/` directory in the `backup-data` bucket of Amazon S3.
 * `--pd`: the option that specifies the Placement Driver (PD) service address.
 * `"${PDIP}:2379"`: the parameter of `--pd`.
 
-> **Note:**
->
-> - When the `local` storage is used, the backup data are scattered in the local file system of each node.
->
-> - It is **not recommended** to back up to a local disk in the production environment because you **have to** manually aggregate these data to complete the data restoration. For more information, see [Restore Cluster Data](#use-br-command-line-to-restore-cluster-data).
->
-> - Aggregating these backup data might cause redundancy and bring troubles to operation and maintenance. Even worse, if restoring data without aggregating these data, you can receive a rather confusing error message `SST file not found`.
->
-> - It is recommended to mount the NFS disk on each node, or back up to the `S3` object storage.
-
 ### Sub-commands
 
 A `br` command consists of multiple layers of sub-commands. Currently, BR has the following three sub-commands:
```

Comment:
version is missing in the following list.

Reply:
Fixed